Makefile

```diff
@@ -45,6 +45,20 @@ vendor:
 vet:
 	$(GO) vet ./...
 
+
+.PHONY: checkdocs
+checkdocs: checkrst checkmd
+
+.PHONY: checkrst
+RST_FILES=$(shell find . -name '*.rst' | grep -v /vendor/ | sort)
+checkrst: $(RST_FILES)
+	rstcheck $^
```

Comment:
Would probably be good to have some type of `command -v` check here, with instructions on how to install these things (or targets to install them into `BIN_DIR` in case of absence).

Reply:
I've added a check in 8e1da49381e49d7104020da967cc1ab988cbeb2b.